Source/System.Management/Pash/ParserIntrinsics/AstBuilder.cs

```diff
@@ -2271,6 +2271,31 @@ CommandParameterAst BuildCommandParameterAst(ParseTreeNode parseTreeNode)
                                            new ScriptExtent(parseTreeNode), requiresArgument);
         }
 
+        IEnumerable<CommandElementAst> PostprocessCommandElements(IEnumerable<CommandElementAst> elements)
+        {
+            var queue = new Queue<CommandElementAst>(elements);
+            while (queue.Count > 0)
```

Comment:
What performance reasons do you have in mind?

While I would write `.Any()`, I deliberately don't have strong coding guidelines in Pash, beyond having tests. So I don't object to `.Count > 0`.